src/libs/actions/BankAccounts.js

```diff
@@ -766,10 +766,6 @@ function setupWithdrawalAccount(data) {
                         console.error(response.message);
                     }
                 }
-
-                if (lodashGet(achData, CONST.BANK_ACCOUNT.VERIFICATIONS.THROTTLED)) {
-                    achData.disableFields = true;
-                }
```

Comment:
This is just dead code pulled in from Web-Secure. I am assuming there we disable fields in some cases, but doesn't seem like we use this anywhere in New Expensify so I removed it.